homeassistant/components/overkiz/cover_entities/generic_cover.py

```diff
@@ -69,6 +69,14 @@ async def async_set_cover_tilt_position(self, **kwargs: Any) -> None:
                 100 - kwargs[ATTR_TILT_POSITION],
             )
 
+    @property
+    def is_opened(self) -> bool | None:
+        """Return if the cover is opened"""
+        if self.current_cover_position is not None:
+            return self.current_cover_position == 100
+        else:
+            return None
```

Comment:
Suggestion for clarity

```suggestion
    def is_opened(self) -> bool | None:
        """Return if the cover is opened"""
        if self.current_cover_position is None:
            return None
        return self.current_cover_position == 100
```